module/VuFind/src/VuFind/ILS/Driver/XCNCIP2.php

```
@@ -163,9 +246,15 @@ protected function sendRequest($xml)
 
         // Process the NCIP response:
         $response = $result->getBody();
+        $this->debug('Got NCIP response: ' . $response);
         $result = @simplexml_load_string($response);
         if (is_a($result, 'SimpleXMLElement')) {
-            $result->registerXPathNamespace('ns1', 'http://www.niso.org/2008/ncip');
+            // If no namespaces are used, add default one and reload the document
+            if (empty($result->getNamespaces())) {
+                $result->addAttribute('xmlns', 'http://www.niso.org/2008/ncip');
+                $result = @simplexml_load_string($result->asXML());
```

Since you're suppressing errors here, do you need to account for the possibility that simplexml_load_string fails and returns false?

---

It should not happen as `asXML` should return well-formed XML string, but things can always fail so I added a test on return value.